tooling/bin/pecl-build

```diff
@@ -6,7 +6,8 @@ rm -f ./bridge/_generated.php
 composer compile
 
 # PECL doesn't like the "-nightly" part of the nightly version number so we have to change it
-dd_version=$(php -r "echo str_replace('1.0.0-nightly', '0.0.0', require 'src/DDTrace/version.php');")
+dd_version=$(cat src/DDTrace/Tracer.php | grep 'const VERSION' | awk '{print $$4}' | cut -d\' -f2 | sed 's/-nightly//g')
```

Comment:
Awk can do all of this.

```suggestion
dd_version=$(awk -F\' '/const VERSION/ {sub(/-nightly/, "", $2); print $2}' < src/DDTrace/Tracer.php)
```

Reply:
Same as above...have no idea what's in it, but it works 😄